widgetastic_manageiq/__init__.py

```diff
@@ -5412,6 +5419,16 @@ class MigrationProgressBar(Widget):
     ERROR_TXT = './/p[contains(@class, "blank-slate-pf-info")]'
     PROGRESS_BARS = './/div[@class="progress-bar"]'
     PROGRESS_DESCRIPTION = './/div[contains(@class,"progress-description")]'
+    ITEM_CUTOVER_BUTTON_LOCATOR = './/button[text()="Schedule Cutover"]'
+    IMMEDIATE_BTN = './/input[contains(@value="schedule_migration_now")]'
+
+    ITEM_SCHEDULE_INPUT_LOCATOR = './/input[@id="dateTimeInput"]'
+    ITEM_MODAL_SCHEDULE_LOCATOR = './/button[text()="Schedule"]'
+    ITEM_MODAL_CANCEL_BUTTON_LOCATOR = './/button[contains(@class,"btn-cancel btn")]'
+    ITEM_MODAL_MINUTE_INCREMENT_LOCATOR = (
+        './/*[@id="dateTimePicker"]/div/div/div[2]/div[1]' "/table/tbody/tr[1]/td[3]"
```

Comment:
Q: can we get better locator for this?

Reply:
I think it is fine unless we gets the proper element.